src/main/java/frc/utils/alerts/Alert.java

```diff
@@ -0,0 +1,36 @@
+package frc.utils.alerts;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Timer;
+import frc.utils.DriverStationUtils;
+import org.littletonrobotics.junction.Logger;
+
+public class Alert {
+
+	public enum AlertType {
+
+		ERROR,
+		PLACEHOLDER,
+		WARNING;
+
+	}
+
+	private static final boolean LOG_TRACE = false;
+	public static final String ALERT_LOG_PATH = "Alerts/";
+	private final AlertType type;
+	private final String logPath;
```

Comment:
public static final
private static final
private final
private

Reply:
no

Reply:
why does logtrace need  too be public?
why does logpath not need to be final? it does not change after class creation and is not static

Reply:
i wrote the order it need to be, not same order different stuff